src/docs/implementations/dynatrace.adoc

```diff
@@ -242,9 +207,50 @@ DynatraceConfig dynatraceConfig = new DynatraceConfig() {
 };
 ----
 
+These properties can also be set in Spring Boot configuration files:
+
+[source,yml]
+----
+management.metrics.export.dynatrace:
+    # Required only if not using the OneAgent endpoint
+    # For SaaS: https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest
+    # For managed deployments: https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest
+    uri: YOUR_METRICS_INGEST_URL
+
+    # should be read from a secure source
+    api-token: YOUR_METRICS_INGEST_TOKEN
+
+    # These properties can only be used with the v2 exporter.
+    v2:
+        # Sets a prefix that is prepended to each exported metric key.
+        metric-key-prefix: my.metric.key.prefix
+
+        # If set to true and a local OneAgent or operator is running, retrieves metadata
+        # and adds it as additional dimensions to all data points (default: true)
+        enrich-with-dynatrace-metadata: true
+
+        # Sets an arbitrary number of key-value pairs as default dimensions.
+        # Micrometer tags will overwrite these dimensions, if they have the same key.
+        # Each exported metric will contain these dimensions.
+        default-dimensions:
+            key1: "value1"
+            key2: "value2"
+
+        # (since 1.9.0) Whether or not to use the Dynatrace-specific summary instruments. (default: true)
+        # This should only be disabled if problems with existing instrumentation are discovered after upgrading to 1.9.0.
+        # Set to false, this will restore the previous (1.8.x) behavior for Timers and DistributionSummaries.
+        use-dynatrace-summary-instruments: true
+
+    # The export interval in which metrics are sent to Dynatrace (default: 60s).
+    step: 60s
+----
+
 For more information about the metadata picked up by the Dynatrace metadata enrichment feature, see https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/enrich-metrics/[the Dynatrace documentation].
 
-In Micrometer 1.9.0, Dynatrace-specific summary instruments (`DynatraceTimer` and `DynatraceDistributionSummary`) were introduced. These specialized instruments are tailored to the Dynatrace metrics ingest, and prevent the creation of invalid metrics. They are available from version 1.9.0 and are used as a drop-in replacement by default. No action is needed from users upgrading to 1.9.0. If there is a discrepancy in the observed metrics, it is possible to return to the previous behavior by setting the `useDynatraceSummaryInstruments` toggle to `false`.
+In Micrometer 1.9.0, Dynatrace-specific summary instruments (`DynatraceTimer` and `DynatraceDistributionSummary`) were introduced.
```

Comment:
I wonder if it's agreed to manually create new lines or keep the whole paragraph in a single line in the source code. Now it's a bit messy with both methods being used in this document.

Reply:
Not sure if anything is enforced (I didn't find anything), but I think that each sentence is on its own line now everywhere in the file.